main.py

```
@@ -124,6 +124,7 @@ async def main():
     # Run startup health checks
     health_manager = HealthCheckManager(bot)
     health_manager.register(OpenAIHealthCheck())
+    health_manager.register(LocaleFileHealthCheck())
```

Flake8 (7.3.0)

[error] undefined name 'LocaleFileHealthCheck'

(F821)